tools/changelog/changelog_test.go

```
@@ -92,6 +94,45 @@ func readFile(t *testing.T, path string) string {
+// long before the mode matters.
+func skipIfStillWritable(t *testing.T, dir string) {
+	t.Helper()
+	probe := filepath.Join(dir, ".probe")
+	f, err := os.Create(probe)
+	if err != nil {
+		if !errors.Is(err, fs.ErrPermission) {
+			t.Fatalf("probing the write denial on %s: %v", dir, err)
+		}
+		return
+	}
+	f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)